### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

Remaining ones: "post_statusses" and [cachable](https://github.com/search?q=repo%3AWordPress%2FWordPress-Coding-Standards%20cachable&type=code)

https://github.com/crate-ci/typos

### CODE REVIEW

File: WordPress/AbstractClassRestrictionsSniff.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ public function process_token( $stackPtr ) {
 	 *
 	 * @return bool
 	 */
-	public function is_targetted_token( $stackPtr ) {
+	public function is_targeted_token( $stackPtr ) {
```
Comment (szepeviktor) [inline WordPress/AbstractClassRestrictionsSniff.php:119]: There is a public method name change here.

Comment (dingo-d) [reply WordPress/AbstractClassRestrictionsSniff.php:119]: From a quick search on the GH there are some external standards which seem to be using this abstract class, so changing this public method would be a major BC break IIRC.

**Review by dingo-d — CHANGES_REQUESTED** [review]
The typos changes in other files are ok, but the changing of the public method would be a BC break so I'm not sure this should be changed.

File: WordPress/AbstractClassRestrictionsSniff.php (hunk 1 above)
Comment (szepeviktor) [reply WordPress/AbstractClassRestrictionsSniff.php:119]: Okay, reverting ...

Comment (szepeviktor) [reply WordPress/AbstractClassRestrictionsSniff.php]: Done 🍏

**Review by dingo-d — APPROVED** [review]

**Review by jrfnl — APPROVED** [review]

**Pull request merged**